packages/playground/src/settings/list-settings.tsx

```diff
@@ -6,7 +6,7 @@ interface SettingsProps {
   setLoadState: (state: string) => void
 }
 
-const ListSettings = ({ loadState, setLoadState }: SettingsProps) => {
+export const PlaygroundListSettings = ({ loadState, setLoadState }: SettingsProps) => {
```

Comment:
do we need this component since it was just for playground?

Reply:
Yes, removed it, but renaming it `PlaygroundListSettings` for better readability.